src/SH12A_ddd/template/detect.dat

```diff
@@ -1,10 +1,9 @@
-*----0---><----1---><----2---><----3---><----4---><----5---><----6--->
-*MSH             -5.0      -5.0       0.0       5.0       5.0      50.0
-*                   1         1       500        -1    ENERGY      emsh
-*----0---><----1---><----2---><----3---><----4---><----5---><----6--->
-*CYL              0.0       0.0       0.0      40.0      7.00      50.0
-*                 400         1       500        -1    ENERGY      ecyl
-*----0---><----1---><----2---><----3---><----4---><----5---><----6--->
-CYL              0.0       0.0       0.0      40.0      7.00      50.0
-                 800         1       500        -1       DDD      dcyl
-*----0---><----1---><----2---><----3---><----4---><----5---><----6--->
+Geometry Cyl
+    Name MyCyl                    ! geometry type
+    R  0.0  40.0   800
+    Z  0.0  50.0   500
+
+Output
+    Filename nb_ddd.bdo
```

Comment:
I'd simply call it `output.bdo`, otherwise DDD files would get strange names, like `nb_ddd_12C.H20....`